publisher/README.md

```diff
@@ -40,7 +40,9 @@ Now initialize Moloco SDK using the provided ad unit ID.
       country_iso: "kr",
       width: 640,
       height: 100,
-      os: "android"
+      os: "android",
+      use_redirect_url: false, // default: false.
+      isLegacy: false, // default: true.
```

Comment:
In my opinion, use_direct_landing and use_endpoint_v1 (default false) sound better to me as the replacing href with final landing URL is not for redirecting but for using deep link to take to the final landing page directly, and legacy sounds somewhat negative to me (and probably to customers) so I would suggest naming it use_endpoint_v1.

Reply:
Thank you for the review!
I agree with your comment. I modified and pushed it.

In addition, I found a bug that always made a request by `originalClickUrl`. I modified it to request only if it is clicked using EventListener.